components/GovernancePower/Vanilla/useDepositCallback.tsx

```diff
@@ -64,9 +70,9 @@ export const useDepositCallback = (
         userAtaPk,
         mint,
         walletPk,
-        transferAuthority.publicKey,
+        publicKeyToUse,
         walletPk,
-        amount
+        amount,
```

Comment:
should this comma be here? seems odd

Reply:
My formatter uses trailing commas, if not wanted I can remove it.

Reply:
huh. surprised that ours doesn't appear to be opinionated on it.